marklogic-data-hub/src/test/java/com/marklogic/hub/security/DataHubSecurityAdminTest.java

```diff
@@ -97,7 +97,7 @@ void createCustomRoleInheritingCertainDataHubRoles() {
         Role customRole = new Role(userWithRoleBeingTestedApi, roleName);
         customRole.setRole(CreateGranularPrivilegesCommand.ROLES_THAT_CAN_BE_INHERITED);
 
-        assertEquals(42, customRole.getRole().size(), "As of DHFPROD-5753 in 5.3.0, 42 roles are expected to be " +
+        assertEquals(43, customRole.getRole().size(), "As of DHFPROD-5753 in 5.3.0, 43 roles are expected to be " +
```

Comment:
Updated the message to reflect this JIRA story and 5.4.0

Reply:
done